### DISCUSSION TITLE
Does Objeck support the union data type?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#324] [General]
Neither Java nor C# supports it. I think Objeck doesn't, too. But what if I want to map C code that use union data type to Objeck?

**Comment by objeck** [comment]
Pointers to C union types can be mapped. Do you need an example?

**Reply by ghost** [reply]
Please give me an example. Thank you.

**Reply by ghost** [reply]
@objeck

**Reply by objeck** [reply]
[Here's](https://github.com/objeck/objeck-lang/blob/master/core/lib/sdl/sdl.cpp#L1039) an example of mapping struct to an Objeck. The same approach can be used to map union types. Many union types are wrapped in structs.
